alacritty_terminal/src/event_loop.rs

```diff
@@ -65,6 +66,7 @@ where
         event_proxy: U,
         pty: T,
         hold: bool,
+        drain_pty: bool,
```

Comment:
Could just pass `PtyOptions` here, but for now the number of arguments isn't too bad so I'm fine either way.

Reply:
We removed `hold`, so no need for this anymore.